src/async.c

```diff
@@ -33,10 +33,14 @@ void xattr_get_execute(napi_env env, void* _data) {
 
   data->value = malloc((size_t) data->value_length);
 
-#ifdef __APPLE__
-  data->value_length = getxattr(data->filename, data->attribute, data->value, (size_t) data->value_length, 0, 0);
+#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__DragonFly__)
+    data->value_length = extattr_get_file(data->filename, EXTATTR_NAMESPACE_USER,
+                                          data->attribute, data->value, (size_t)data->value_length);
+    printf("async.c: %s : data->value_length: %zd \n", __func__, data->value_length);
```

Comment:
I don't think you meant to include the printfs here and elsewhere

Reply:
Yes, i will remove this debug prints